sanic.py

```diff
@@ -1,14 +1,22 @@
 from discord.ext import commands
 
-from sanicbot.core.cogs import HelpCog, GitCog
 from sanicbot.core.config import config
 
-bot = commands.Bot(help_command=None, command_prefix=None)
-git_cog = GitCog(bot)
 
+EXTENSIONS = (
+    'sanicbot.extensions.git',
+    'sanicbot.extensions.help',
+)
+
+
+def main():
+    bot = commands.Bot(help_command=None, command_prefix='!')
+
+    for ext in EXTENSIONS:
+        bot.load_extension(ext)
```

Comment:
By adding Cog classes using `bot.add_cog` we can avoid adding setup() functions to the cog files. (see PR #8 for implementation). By extending the `commands.Bot` class we have more control over the bot scope and can handle adding cogs more elegantly.